htsget.md

```diff
@@ -239,8 +239,9 @@ The client can request only variants overlapping a given genomic range. The resp
 `id`  
 _required_
 </td><td>
-Study ids from which variants are to be returned.
+A string specifying which variants to return.
```

Comment:
Should we use 'identifier'? That would be more general than 'Study ID' but more specific than 'string'?

Reply:
Agree, it just replicated what was already there (this refers to an old commit btw).